### PR TITLE
Add nil-check for loki.enable

### PR DESCRIPTION
**Pull request by jotak**

### CODE REVIEW

File: controllers/flowlogspipeline/flp_common_objects.go [hunk 1]
```diff
@@ -439,7 +439,7 @@ func (b *builder) addTransformStages(stage *config.PipelineBuilderStage) (*corev
 	})
 
 	// loki stage (write) configuration
-	if *b.desired.Loki.Enable {
+	if helper.PtrBool(b.desired.Loki.Enable) {
```
Comment (jotak) [inline controllers/flowlogspipeline/flp_common_objects.go:442]: I'm not exactly sure when Enable can be nil (upgrade scenario?) - but we should actually consider nil as true, to keep consistency with the default behaviour. So I'll write a new helper method instead

Comment (jotak) [reply controllers/flowlogspipeline/flp_common_objects.go]: (done)

**Review by jpinsonneau — APPROVED** [review]
thanks @jotak !

**Pull request merged**